code/sample-app/app.manifest

```diff
@@ -0,0 +1,7 @@
+# ATTRIBUTES
+
+# METHODS
+function run()
```

Comment:
run -> entry?

Reply:
What "framework" provides
- The PATH to application itself
- App-local storage
- Load Standard(?) libraries

CMAKE_CURRENT_SOURCE_DIR and CMAKE_CURRENT_BINARY_DIR